src/layer/oneway.js

```diff
@@ -10,6 +10,7 @@ export const surface = {
     ["==", ["get", "oneway"], 1],
     ["!=", ["get", "ramp"], 1],
     ["!", ["in", ["get", "brunnel"], ["literal", ["bridge", "tunnel"]]]],
+    ["!", ["in", ["get", "class"], ["literal", ["path", "track"]]]],
```

Comment:
Nit: I find this somewhat more readable, though I don’t think it would affect performance:

```suggestion
    ["match", ["get", "brunnel"], ["bridge", "tunnel"], false, true],
    ["match", ["get", "class"], ["path", "track"], false, true],
```